### ISSUE TITLE
The difference between 85.96 and 85.66 original MAE accuracy

### ISSUE THREAD

**Issue by LiuDongyang6** [facebookresearch/ToMe#17]
Thank you for this work!

I observe that two different numbers (85.96 and 85.66) are reported for MAE ViT-L/16 original accuracy, and I wonder what the underlying difference is. 

 Thanks!

**Comment by dbolya** [comment]
Thanks for catching that.
The 85.66 is a model we trained ourselves, while the 85.96 is the model available from the [official pytorch MAE repo](https://github.com/facebookresearch/mae/blob/main/FINETUNE.md). Since our training setup couldn't reproduce the 85.96 number, we had to train our own model for fair comparison.

**Comment by LiuDongyang6** [comment]
Thanks
